manual/adobeshockwaveplayer/README.md

```diff
@@ -0,0 +1,9 @@
+﻿# ![Adobe Shockwave Player Logo](https://cdn.jsdelivr.net/gh/pauby/ChocoPackages@master/icons/adobeshockwaveplayer.svg "Adobe Shockwave Player") [Adobe Shockwave Player](https://chocolatey.org/packages/adobeshowaveplayer)
```

Comment:
There is a typo in the package name URL at the end.

Reply:
done